src/HttpRequest.cc

```diff
@@ -738,3 +738,31 @@ HttpRequest::manager(const CbcPointer<ConnStateData> &aMgr, const AccessLogEntry
     }
 }
 
+const Ip::Address *
```

Comment:
This function will need a Doxygen description once its public declaration is gone.